src/shared/components/image-button/ImageButton.tsx

```diff
@@ -24,52 +28,57 @@ import Tooltip from 'src/shared/components/tooltip/Tooltip';
 
 import { Status } from 'src/shared/types/status';
 
-import imageButtonStyles from './ImageButton.scss';
+import styles from './ImageButton.scss';
 
 export type ImageButtonStatus =
   | Status.DEFAULT
   | Status.SELECTED
   | Status.UNSELECTED
   | Status.DISABLED;
 
-export type Props = {
-  status: ImageButtonStatus;
-  description: string;
+type ChildlessButtonProps = Omit<
+  ButtonHTMLAttributes<HTMLButtonElement>,
+  'children'
+>;
+
+export type Props = ChildlessButtonProps & {
+  status?: ImageButtonStatus;
+  description?: string;
   image: FunctionComponent | string;
-  className?: string;
-  statusClasses?: { [key in ImageButtonStatus]?: string };
-  onClick?: (event?: React.MouseEvent<HTMLDivElement>) => void;
 };
 
 export const ImageButton = (props: Props) => {
+  const {
+    status = Status.DEFAULT,
+    description,
```

Comment:
should description be `description = ''` for the default value because what will happen if no description is passed. will it not be undef instead of empty string?

Reply:
Both undef and '' empty string will not show the alt even though accessibility reason there should always be an alt but thats not applicable to us for now.